# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/newtab/content-src/lib/init-store.js
# Commit: 7ec776d31a2c
# Full Hash: 7ec776d31a2c6e0ffc1e5ee311d63993317ea7ca
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2020-04-17 21:46:50
# Regressor Bug: 1614502
# File Overlap Count: 1
# Description:
#   Bug 1614502 - Expose utility functions for rendering Activity Stream. r=Mardak
#   
#   These will be used by a later patch inside of a ChromeWorker using
#   ReactDOMServer.
#   
# ==============================================================================

diff -r 8bf3fe111c40 -r 7ec776d31a2c browser/components/newtab/content-src/lib/init-store.js
--- a/browser/components/newtab/content-src/lib/init-store.js	Fri Apr 17 15:31:06 2020 +0000
+++ b/browser/components/newtab/content-src/lib/init-store.js	Fri Apr 17 15:31:11 2020 +0000
@@ -135,9 +135,10 @@
  * @param  {object} intialState (optional) The initial state of the store, if desired
  * @return {object}          A redux store
  */
-export function initStore(reducers) {
+export function initStore(reducers, initialState) {
   const store = createStore(
     mergeStateReducer(combineReducers(reducers)),
+    initialState,
     global.RPMAddMessageListener &&
       applyMiddleware(
         rehydrationMiddleware,